lib/bin/s3-create-bucket.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const Minio = require('minio');
+
+const { server, bucketName, accessKey, secretKey } = require('config').get('default').s3blobStore;
+
```

Comment:
Can this all be moved back to the `Makefile`?  This might make it less tempting for end-users, and should make it clearer that an ideal S3 production credential would not have authority to create buckets.

Reply:
There's not obviously a simpler way to do this which supports both CI & local, and doesn't require more dependencies.

Discussed at:

* https://github.com/minio/minio/issues/4769
* https://stackoverflow.com/questions/71827591/how-to-create-default-bucket-for-minio-within-github-actions-using-minioedge-ci
etc